### PR TITLE
RATEST-79:Failing Test:XXS On Phone number field test

### PR DESCRIPTION
**Pull request by gracebish**
Issue Worked on
https://issues.openmrs.org/browse/RATEST-79
What i changed
Fix legacy code by removing @ignore annotation

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java [hunk 1]
```diff
@@ -36,7 +35,6 @@ public void setup() {
     }
 
     @Test
-    @Ignore
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java:39]: you need to redo this and send through a clean PR without that trailing change i trashed

**Comment by kdaud** [comment]
@gracebish have you had a look at the conflicts with your PR. Kindly address this as well and update the commit

File: ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java (hunk 1 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/XSSOnPhoneNumberFieldTest.java:39]: @gracebish did you address the comment above

**Comment by gracebish** [comment]
I am closing this pr

**Comment by gracebish** [comment]
I would like to push through a clean pr

**Pull request closed without merging**